gson/src/test/java/com/google/gson/functional/FormattingStyleTest.java

```diff
@@ -89,6 +90,13 @@ public void testNewlineOs() {
     assertEquals(EXPECTED_OS, json);
   }
 
+  @Test
+  public void testCompact() {
+    Gson gson = new GsonBuilder().setPrettyPrinting(null).create();
+    String json = gson.toJson(INPUT);
+    assertEquals(EXPECTED_COMPACT, json);
```

Comment:
You can use Truth

```java
assertThat(json).isEqualTo(EXPECTED_COMPACT)
```

_Note: Some other above tests use `assertEquals(...)`, should be modified them too._

Reply:
Good point, have changed it